project/Dependencies.scala

```diff
@@ -17,7 +17,7 @@ object Dependencies {
 
 
   private val workbenchLibsHash = "d619ea2a-SNAP"
```

Comment:
update `workbenchLibsHash` instead....it's easier when all moduels are on the same hash